src/main/java/seedu/address/model/patient/exceptions/PersonIsNotPatient.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.patient.exceptions;
+
+/**
+ * Signals the operation that a person is not a patient instance.
+ */
+public class PersonIsNotPatient extends RuntimeException {
```

Comment:
A convention to follow when naming custom Exceptions would be to append 'Exception' to the end of the class name